### PR TITLE
Changing demo-swarm to demo-legacy-swarm

### PR DESCRIPTION
**Pull request by neelimamukiri**

### CODE REVIEW

File: websrc/source/documents/networking/l2-vlan.md [hunk 1]
```diff
@@ -78,16 +78,16 @@ Log in to containers and verify the IP address has been allocated from the netwo
...
+f864841d626d        alpine                         "sh"                8 minutes ago       Up 8 minutes                            contiv-demo-legacy-swarm-l2-6/agitated_brahmagupta
+9f3aed39e95b        alpine                         "sh"                9 minutes ago       Up 8 minutes                            contiv-demo-legacy-swarm-l2-6/happy_pasteur
+a3111bb0c9e1        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-4/etcd
+6881c4c1b380        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-6/etcd
+a8eb4c23c009        quay.io/coreos/etcd:v2.3.1     "/etcd"             10 minutes ago      Up 10 minutes                           contiv-demo-legacy-swarm-l2-5/etcd
+b54850f408d8        skynetservices/skydns:latest   "/skydns"           5 weeks ago         Up 10 minutes       53/tcp, 53/udp      contiv-demo-legacy-swarm-l2-6/defaultdns
```
Comment (rhim) [inline websrc/source/documents/networking/l2-vlan.md]: I have a feeling that these skydns containers are bogus as well.

Why don't you leave out the documents outside of tutorials for now. We can fix those in another PR.

Comment (gaurav-dalvi) [inline websrc/source/documents/networking/l2-vlan.md]: yes we dont use skydns anymore.

**Comment by neelimamukiri** [comment]
Done, I removed the non tutorial stuff.

**Comment by rhim** [comment]
LGTM.

**Pull request merged**